ICAFIX/ApplyHandReClassifications.sh

```diff
@@ -125,10 +129,21 @@ log_Msg "OriginalFixSignal: ${OriginalFixSignal}"
 OriginalFixNoise="${FIXFolder}/Noise.txt"
 log_Msg "OriginalFixNoise: ${OriginalFixNoise}"
 
-ReclassifyAsSignal="${ResultsFolder}/ReclassifyAsSignal.txt"
+#handle defaults that rely on other parameters
+if [[ "$ReclassifyAsSignal" == "" ]]
+then
+	ReclassifyAsSignal="${ResultsFolder}/ReclassifyAsSignal.txt"
+else
+	ReclassifyAsSignal="${ResultsFolder}/${ReclassifyAsSignal}"
```

Comment:
I would substantially prefer that absolute paths work for any option that takes a filename.  Please put the `$ResultsFolder` location for these arguments into the ApplyAutoReclean script, rather than having this script add it to any supplied filenames.  The only reason `$ResultsFolder` is in these special default tests is to preserve previous behavior, where the default location depends on `$Subject` and `$StudyFolder` (and therefore can't be handled by newopts static defaults).